src/server/controllers/user.js

```diff
@@ -1,39 +1,63 @@
-import bcrypt from 'bcrypt';
-import jwt from 'jsonwebtoken';
-import { PrismaClient } from '@prisma/client'
+import bcrypt from "bcrypt";
+import jwt from "jsonwebtoken";
+import { PrismaClient } from "@prisma/client";
 const prisma = new PrismaClient();
 
-const jwtSecret = 'mysecret';
+const jwtSecret = "mysecret";
```

Comment:
This would be good as an environment variable. Otherwise, someone could gain access to your code and then would be able to decode your tokens